examples/atlas-advanced-cluster/global-cluster/README.md

```diff
@@ -0,0 +1,60 @@
+# MongoDB Atlas Provider -- Global Cluster 
+This example creates a project and a Global Cluster.
+
+
+## Dependencies
+
+* Terraform MongoDB Atlas Provider v1.10.0
+* A MongoDB Atlas account 
+
+```
+Terraform >= 0.13
++ provider registry.terraform.io/terraform-providers/mongodbatlas v1.10.0
+```
+
+
+## Usage
+**1\. Ensure your MongoDB Atlas credentials are set up.**
```

Comment:
Can you also add option for users to Authenticate with AWS Secrets Manager - https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/index.html.markdown#aws-secrets-manager

Reply:
updated